docs/site-example/i18n/en.po

```diff
@@ -13,15 +13,22 @@ msgstr ""
 msgid "gluon-config-mode:welcome"
 msgstr ""
 "Welcome to the setup wizard of your new Freifunk Duckburg node. "
-"Please fill out the following form and submit it."
+"Please fill out the following form and submit it. "
+"The name of this node is the visible name on the Freifunk map. It is different "
+"from the WiFi Network SSID of the Freifunk Network that has a fixed name. "
+"You can choose any name, so think of something descriptive or beautiful."
```

Comment:
I think this text should rather be added as a help text for the hostname field, not to the welcome text.